main_routers/cookies_login_router.py

```
@@ -0,0 +1,229 @@
+    # 安全检查文件对象是否存在
+    if not cookie_file or not cookie_file.exists():
+        return {"success": True, "message": f"{platform} 凭证本就不存在"}
+            
+    try:
+        cookie_file.unlink()
+        key_file = CONFIG_DIR / f"{platform}_key.key"
+        if key_file.exists():
+            key_file.unlink()
+
+        return {"success": True, "message": f"✅ {platform.capitalize()} 凭证已物理粉碎"}
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)